js/options.js

```diff
@@ -24,7 +24,16 @@ function save_options() {
 	var spamCount = document.getElementById('SpamCount').value;
 	var blockSpam = document.getElementById('BlockSpam').checked;
 	var omniSearch = document.getElementById('OmniSearch').checked;
+	//Quni Ticket Breakdown
 	var quniTickets = document.getElementById('QuniTickets').checked;
+	var siQueue = document.getElementById('SIQueue').checked;
+	var taQueue = document.getElementById('TAQueue').checked;
+	var three60Queue = document.getElementById('360Queue').checked;
+	var eeQueue = document.getElementById('EEQueue').checked;
+	var themesQueue = document.getElementById('ThemesQueue').checked;
+	var vocQueue = document.getElementById('VocQueue').checked;
+	var statQueue = document.getElementById('StatQueue').checked;
+	var intQueue = document.getElementById('IntQueue').checked;
```

Comment:
Definitely could use more descriptive names, but folding them into an object seems to break the style of how every other variable is set from lines 2-154 of this file.

Also would not reduce lines of code, at least the way I'm imagining.

Do you have an example of what lines 29-36 could look like instead?